tests-integration/tests/grpc.rs

```diff
@@ -129,6 +136,7 @@ pub async fn test_grpc_message_size_ok(store_type: StorageType) {
     let config = GrpcServerConfig {
         max_recv_message_size: 1024,
         max_send_message_size: 1024,
+        tls: TlsOption::default(),
```

Comment:
I think we can use `..Default::default()` here to save effort if we are adding more members in future.